product/juno/module/juno_reset_domain/include/mod_juno_reset_domain.h

```diff
@@ -0,0 +1,74 @@
+/*
+ * Arm SCP/MCP Software
+ * Copyright (c) 2020, Arm Limited and Contributors. All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Description:
+ *     JUNO Reset Domain module.
+ */
+
+
+#ifndef MOD_JUNO_RESET_DOMAIN_H
+#define MOD_JUNO_RESET_DOMAIN_H
+
+#include <fwk_id.h>
+
+#include <stdint.h>
```

Comment:
swap and remove empty line

Reply:
Removed the line. However kept the order as it is, just to be consistent with other module files.